tests/improver-ensemble-calibration/03_trunc_gaussian.bats

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bats
```

Comment:
In general, this test needs to be changed to test `truncated gaussian` rather than `gaussian`, which I think is the aim of the test.